submissions/mxmgny/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+let cat = {
+    appearance: "cat",
+    name: "Garfield",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    shout: "I love lazagna!",
+    friends:['Selina', 'Jeremy']
+};
+let cat_woman = {
+    appearance:"cat-woman",
+    name:"Selina",
+    legs:2,
+    hands:2,
```

Comment:
Also code style. Everywhere keep one space after `key:` as you did that before.

Reply:
Done.